docs/settings/reporting-settings.asciidoc

```diff
@@ -27,8 +27,13 @@ setting to preserve the same key across multiple restarts and multiple instances
 ==== Kibana server settings
 
 Reporting uses the Kibana interface to generate reports. In most cases, you don't need
-to configure Reporting to communicate with Kibana. However, if you use a reverse-proxy
-to access Kibana, you must set the proxy port, protocol, and hostname.
+to configure Reporting to communicate with Kibana. 
+
+If you use a reverse-proxy to access Kibana, you can set the proxy port, protocol, 
+and hostname to have Reporting use the proxy to load Kibana URLs. In most cases, 
+this is only required if the proxy is responsible for modifying requests to 
+make them work with Kibana. For example, a reverse proxy can remove a 
+base from requests so Kibana will properly serve the requests.
```

Comment:
Let's change this to:
> For example, a reverse proxy can be responsible for translating the public host name for the Kibana server to a different name in a private network. It may also be responsible for maintaining encrypted communications between the outside world and hosts in the internal network. Encrypted communications will fail if the host name from the request doesn't match the destination server's host name, because of being rejected by the proxy. In such cases, Reporting should not assume a direct connection to the Kibana server: an administrator must configure Reporting use send the hostname the proxy expects, which is done with the `xpack.reporting.kibanaServer.*` settings.

That setup is probably more common than rewriting the path to remove a base path.